docs/cmd/md/meroxa_config.md

```diff
@@ -0,0 +1,28 @@
+## meroxa config
+
+Manage your Meroxa CLI configuration
+
+```
+meroxa config [flags]
+```
+
+### Options
+
+```
+  -h, --help   help for config
+```
+
+### Options inherited from parent commands
+
+```
+      --config string      config file
+      --debug              display any debugging information
+      --json               output json
+      --timeout duration   set the client timeout (default 10s)
```

Comment:
I'm unsure if there's a specific format the docs usually follow, but would it also make sense to do something like the following?

```suggestion
      --timeout number  set the duration of the client timeout in seconds (default 10s)
```

Reply:
@jayjayjpg that's a nice suggestion, thank you. I'll change it [in the implementation](https://github.com/meroxa/cli/blob/master/cmd/meroxa/global/global.go#L50) (docs are autogenerated)